modules/optionals/cloud9/README.md

```diff
@@ -0,0 +1,50 @@
+# Cloud9 Module
+
+## Description
+
+This module creates a Cloud9 instance.  It is not required module.
+
+This module:
+- creates a Cloud9 instance
+  - Resizes the root volume total storage capacity
+
+##### Please provide `deployspec.yaml` and `modulestack.yaml` files which will be consumed by SeedFarmer CLI.
+
+## Inputs/Outputs
+
+### Input Paramenters
+
+#### Required
+
+- `instance_type`: Type of instance to launch
+  - default: t3.small
+
+#### Optional
+
+- `image_id`: The identifier for the Amazon Machine Image (AMI) that's used to create
+the EC2 instance. To choose an AMI for the instance, you must specify a
+valid AMI alias or a valid AWS Systems Manager path
+  - default: ami-0beaa649c482330f7
```

Comment:
Since AMIs are region locked, we should support both mechanisms, lookup of the AMI from SSM or AMI directly with resolution mechanism:

If AMI specified directly, use that
If no AMI specified but SSM key for AMI specified lookup AMI via SSM
If no AMI and no SSM Key specified, default to Ubuntu Desktop AMI (Same one used by Cloud9 when setting up via AWS Console UI)

Reply:
the cloud9 construct only supports AMI aliases and SSM paths, which auto-resolves the ami to use based on the region. Also, only three images are supported for cloud9, i've documented them in the README and other places where errors will be raised if the wrong image is used. 

I've added a function to do a lookup for a SSM path if an ami_id is passed and matches a supported image, but i did not document that. I document in the README to simply use an alias or SSM path, i think that is a better user experience @dgraeber